packages/status-js/src/client/client.ts

```diff
@@ -161,7 +161,7 @@ class Client {
              * >@see https://forum.vac.dev/t/waku-v2-scalability-studies/142/2
              */
             bootstrap({
-              list: peers[environment],
+              list: peers['production'],
```

Comment:
Since we are passing hardcoded string `production`, do we need to keep the `environment` key?

Reply:
ditto https://github.com/status-im/status-web/pull/543#discussion_r1537111217